org.eclipse.dartboard/plugin.xml

```diff
@@ -174,7 +174,7 @@
        <launchGroup
              bannerImage="platform:/plugin/org.eclipse.debug.ui/icons/full/wizban/run_wiz.png"
              id="org.eclipse.dartboard.launchGroup"
-             image="platform:/plugin/org.eclipse.debug.ui/icons/full/etool16/run_exc.png"
+             image="assets/img/dart_16.png"
```

Comment:
the eclipse conventional directory name is "icons"

Reply:
Done.